generators/base/logic/options.mjs

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2013-2022 the original author or authors from the JHipster project.
+ *
+ * This file is part of the JHipster project, see https://www.jhipster.tech/
+ * for more information.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Render content
+ *
+ * @param {object} generator reference to the generator
+ * @param {object} options options
+ */
+const appendYeomanOptionsFromGeneratorOptions = (generator, options) => {
+  return {
+    root: options.root || generator.jhipsterTemplatesFolders || generator.templatePath(),
+    context: generator,
+    ...options,
+  };
+};
```

Comment:
There is no reason for this.

Reply:
I just moved the code, I don't know if it's useful or not but it was there... I didn't want to introduce regresion. If not needed we'll remove it later

Reply:
always no idea of what it does: leaving it here for further investigation (will be easier now that I can debug with vscode, thanks to you^^)